src/nginx.go

```diff
@@ -72,8 +93,14 @@ func parseStatusURL(statusURL string) (hostname, port string, err error) {
 		return
 	}
 
+	isHTTP := u.Scheme == httpProtocol || u.Scheme == httpsProtocol
+	if !isHTTP {
+		err = errors.New("unsupported protocol scheme")
+		return
```

Comment:
you are not handling error. This is the main function. I guess what you want is:  `fatalIfErr(err)`